### PR TITLE
chore: use web test runner

### PR DESCRIPTION
**Pull request by LarsDenBakker**
Testing out web test runner on lion, making a PR to test the CI.

There are test failures, they're doing some pretty fancy DOM stuff and probably relied on some environment in Karma that's not there here. Would be good to get some extra 👀 on that if anyone has some time to spare :)

### CODE REVIEW

**Comment by jorenbroekema** [comment]
I'll have time to look into this tomorrow afternoon I think :) should be able to help at least with those nested overlays tests and button tests

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "@open-wc/testing-karma-bs": "^1.3.30",
```
Comment (daKmoR) [inline package.json:42]: should be removed right?
- `@open-wc/testing-karma`
- `@open-wc/testing-karma-bs`

Comment (daKmoR) [reply package.json:42]: also karma.config files

Comment (jorenbroekema) [reply package.json:42]: also what do we do with test:bs and test:prune-snapshots and test:update-snapshots

Comment (daKmoR) [reply package.json:42]: we don't use browserstack anyways... so remove

do we have failing snapshots tests? are we using shapshots? if no then remove

**Comment by tlouisse** [comment]
Looks promising 💪 

It would be nice if we also fix all 'collateral' errors (so not failing tests, but uncaught errors). 
My guess is that we can do this by guarding/rejecting 'outstanding Promises' on disconnectedCallback...

**Comment by jorenbroekema** [comment]
Would it be acceptable to do it in a separate PR? and get this merged. I'm getting a little tired of karma for multiple reasons 🙈

**Comment by tlouisse** [comment]
> Would it be acceptable to do it in a separate PR? and get this merged. I'm getting a little tired of karma for multiple reasons 🙈

Would be fine with me. Only generates a lot of 'noise' in your console and ci now.
So that's why we need to fix it soon I think.
Will make a task for that...

**Comment by tlouisse** [comment]
One more question @jorenbroekema / @LarsDenBakker : will it be possible still to use Browserstack later?

**Comment by LarsDenBakker** [comment]
@tlouisse yep: https://github.com/modernweb-dev/web/tree/master/packages/test-runner-browserstack

**Pull request merged**